src/patternfly/upgrade-examples/AlertUpgradeExamples/examples/alert-upgrade-example-pf4-notification.hbs

```diff
@@ -0,0 +1,63 @@
+{{#> alert-upgrade-examples}}
+{{#> alert alert--modifier="pf-m-success" alert--attribute='aria-label="Success Notification"'}}
+  {{#> alert-icon alert-icon--success="true"}}
+  {{/alert-icon}}
+  {{#> alert-body alert-body--attribute='role="alert"'}}
+    {{#> alert-description}}
+			The server configuration changed.
+		{{/alert-description}}
+	{{/alert-body}}
+  {{#> alert-action}}
+		{{#> button btnClass="pf-c-button" btnAttributes='aria-label="Remove"'}}
```

Comment:
for the `button` component's partial, `.pf-c-button` is included automatically. But you need to add a modifier class. So just change this (and the other buttons in the notification demo examples) from `btnClass="pf-c-button"` to `btnClass="pf-m-plain"`

Reply:
fixed, thanks for finding that